vars-research.yml

```diff
@@ -12,11 +12,14 @@ govuk-notify-enquiry-submitted-to-tp-template-id: b24ffca7-6119-4dbf-8ae2-dceb2c
 govuk-notify-enquiry-response-received-confirmation-to-enquirer-template-id: f64db52d-08ce-4f2a-9b37-a374db1ad77a
 govuk-notify-enquiry-response-submitted-confirmation-to-tp-template-id: fa4f3699-4e98-4cea-add3-ac20ff05a0ca
 govuk-notify-email-verification-template-id: badaacff-87b8-4046-b953-a85a0a9e2a8b
-override-email-address: 
-email-amalgamate-responses: true
+email-override-email-address: 
+email-merge-responses: true
 email-all-sent-to-enquirer: true
 feature-flag-verify-email: true
 feature-flag-enquiry-builder: true
+feature-flag-send-emails-from-ntp: true
+feature-flag-verify-email: true
```

Comment:
Can we remove this? As we already have it.

Reply:
Good spot - sorted